### PR TITLE
error on line 114

### PR DESCRIPTION
**Pull request by divyanshu1593**
In signIn.html on line 114 there is fetch request which is not working. but similar fetch request is working fine in signInDemo.html (code is same). so when I tried to debug on chrome it is giving error on line 114. the server is getting the request (tried console loging it and it is working fine on server side).

### CODE REVIEW

File: signIn.html [hunk 1]
```diff
@@ -83,7 +83,7 @@ <h1 class="heading"><span>SCALE</span>TECH</h1>
            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR7H0u13wFrskVIje5pLMN3kCExmMAQzYOKaQwMYiHNT03LUJIiaoCkTBPqjwRpund-0hU&usqp=CAU" alt="scaletech logo" height="400px" width="400px">
         </div>
         <div class="details">
-            <form action="#" method="POST" >
+            <form action="" method="POST" onsubmit="sendData()">
```
Comment (JeelPatel231) [inline signIn.html]: Here you must pass the event explicitly to handle that in javascript code, and the handler code should return a boolean for validation.

onsubmit="return sendData(event)"

File: signIn.html [hunk 2]
```diff
@@ -96,8 +96,32 @@ <h1>Sign In</h1>
                 <p>Don't Have account?<a href="signup.html" style="text-decoration: none; color:rgba(37, 173, 208, 0.94);"> Create Account</a></p>
 
             </form>
+                        
         </div>
     </div>
 </div>
+<script>
+    async function sendData(){
```
Comment (JeelPatel231) [inline signIn.html]: here you get the event passed from onsubmit event and prevent the default behavior using event.preventDefault()

**Comment by divyanshu1593** [comment]
👍

File: signIn.html (hunk 1 above)
Comment (divyanshu1593) [reply signIn.html]: 👍

**Pull request merged**